firmware/shared/controls/DI_driverTorqueRequest_test.cc

```diff
@@ -10,19 +10,16 @@ int main() {
     // Write your test cases.
 
     // should return 0
-    ASSERT_CLOSE(ctrl::DI_driverTorqueRequest(false, true, 2.5), 0.0);
+    ASSERT_CLOSE(ctrl::DI_driverTorqueRequest(false, true, 2.5), 0);
 
-    // should return AccelPedPos = 1 (edge case)
-    ASSERT_CLOSE(ctrl::DI_driverTorqueRequest(true, false, 2.5), 1.0);
+    // should return AccelPedPos = 2.5
+    ASSERT_CLOSE(ctrl::DI_driverTorqueRequest(true, false, 2.5), 2.5);
 
-    // should return round(AccelPedPos) = 1
-    ASSERT_CLOSE(ctrl::DI_driverTorqueRequest(true, false, 0.6f), 1.0f);
+    // should return AccelPedPos = 0.0006f
+    ASSERT_CLOSE(ctrl::DI_driverTorqueRequest(true, false, 0.0006f), 0.0006f);
 
-    // should return AccelPedPos = 0
-    ASSERT_NCLOSE(ctrl::DI_driverTorqueRequest(true, false, 0.4), 2.0);
-
-    // should return AccelPedPos = 0 (negative position case)
-    ASSERT_EQ(ctrl::DI_driverTorqueRequest(true, false, -1), 0);
+    // should return AccelPedPos = 0.4
+    ASSERT_NCLOSE(ctrl::DI_driverTorqueRequest(true, false, 0.4), 0.5);
 
```

Comment:
Can you add a test with boolean parameters `true, true`? It's the only bool combination not tested